src/services/userServices/userService.js

```diff
@@ -0,0 +1,35 @@
+import api from '../../api/api'
+import {routs} from '../../constants/constants'
+
+export const userService = {
+  login,
+  register,
+  checkEqualEmail,
+  logout
+}
+
+function login(email, password) {
+  return api.postRequestWithoutToken(routs.login, {email, password}).then(
+    user => {
+      const {token, ...userData} = user
+      localStorage.setItem('token', token)
+      localStorage.setItem('user', JSON.stringify(userData))
```

Comment:
We have a lot of records of "localStorage.setItem('token', token) (or getItem('token')". I think, it's better to write some separate service with get/set  token functions, inject it and use it after everywhere in project. Also, we could move "Verify token" logic at this service.